borg/fuse.py

```diff
@@ -54,6 +55,7 @@ def __init__(self, key, repository, manifest, archive, cached_repo):
         self.pending_archives = {}
         self.accounted_chunks = {}
         self.cache = ItemCache()
+        self.data_cache = LRUCache(capacity=20, dispose=id)
```

Comment:
that's up to 160MB (which is significant for smaller devices like rpi).
it sounded like a major speedup could be already achieved with 1 (or few) cached chunks.